typed-racket-lib/typed-racket/base-env/base-types-extra.rkt

```diff
@@ -17,7 +17,7 @@
 (define-other-types
   -> ->* case-> U Rec All Opaque Vector
   Parameterof List List* Class Object Values Instance Refinement
-  pred Struct Struct-Type Prefab Top Bot)
+  pred Struct Struct-Type Prefab Top Bot Distinction)
```

Comment:
I guess I'm confused why the work done by this `define-type-alias` form can't be done manually in `define-new-subtype/pass1`.